app/assets/javascripts/modal/modal-workflow.js

```diff
@@ -34,6 +35,14 @@ ModalWorkflow.prototype.renderError = function (result) {
   window.Raven.captureException(result)
   console.error(result)
   this.$multiSectionViewer.showStaticSection('error')
+  this.setModalLabel()
+}
+
+ModalWorkflow.prototype.setModalLabel = function () {
+  let sectionHeading = this.$multiSectionViewer.querySelector('h1')
```

Comment:
I think I've asked this before, but can't remember the answer - is there a reason we're not using `const`?
And just to double check, this will get transpiled to `var`, right?

Reply:
No, that should be a `var` – no transpilation at the moment. Good spot, will update.